git/git.go

```diff
@@ -237,10 +237,10 @@ func LsTree(ref string) (*subprocess.BufferedCmd, error) {
 func ResolveRef(ref string) (*Ref, error) {
 	outp, err := gitNoLFSSimple("rev-parse", ref, "--symbolic-full-name", ref)
 	if err != nil {
-		return nil, fmt.Errorf("Git can't resolve ref: %q", ref)
+		return nil, fmt.Errorf("git can't resolve ref: %q", ref)
 	}
 	if outp == "" {
-		return nil, fmt.Errorf("Git can't resolve ref: %q", ref)
+		return nil, fmt.Errorf("git can't resolve ref: %q", ref)
```

Comment:
In these two cases, I think we're referring to the project Git, not the binary `git`, so keeping these upper case is probably better.

Reply:
OK, reverted.